cluegonauts/clueless/templates/clueless/gameb.html

```diff
@@ -35,6 +35,11 @@ <h2 class="card-title">{{ location.name }}</h2>
                         {% else %}
                             <div id="{{ location.location_id }}_card" class="card py-20 rounded-lg bg-blue-500 opacity-55 w-auto inline items-center" role="button">
                                 <div class="card-body" id="{{ location.location_id }}">
+                                    {% if location.image %}
```

Comment:
I'd recommend using a layout like [this](https://daisyui.com/components/card/#card-with-image-overlay) so the image is the whole box (wrap in <figure> element), we don't need the blue grid boxes anymore. Would also be good to fix sizing/ layout, the game board appears stretched out right now: 
![image](https://github.com/user-attachments/assets/8e32f259-fb96-48ab-aaab-5b523ce60baf)